src/app/admin/dashboard/tabele/page.tsx

```
@@ -511,67 +644,114 @@ export default function AdminTablesCleanPage() {
-                  <p className="text-gray-600">Učitavam tabele...</p>
+                  <p className="text-sm sm:text-base text-gray-600">
+                    Učitavam tabele...
+                  </p>
                 </div>
               </div>
             ) : (
-              <table className="min-w-full divide-y divide-gray-200">
+              <table
+                className="min-w-full divide-y divide-gray-200"
+                role="table"
+                aria-label={`${currentLeague?.name} tabela`}
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<table>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)